tensorflow_datasets/core/dataset_info.py

```diff
@@ -407,7 +408,53 @@ def initialize_from_bucket(self):
       out_fname = os.path.join(tmp_dir, os.path.basename(fname))
       gcs_utils.download_gcs_file(fname, out_fname)
     self.read_from_directory(tmp_dir)
-
+  
```

Comment:
Please add a test, even if you do not check that it is actually displaying something, it is important to check that the code is not raising error.